torch/csrc/autograd/python_engine.cpp

```diff
@@ -225,6 +226,55 @@ PyObject *THPEngine_run_backward(THPEngine *self, PyObject *args, PyObject *kwar
   END_HANDLE_TH_ERRORS
 }
 
+PyObject *THPEngine_run_forward(THPEngine *self, PyObject *args, PyObject *kwargs)
+{
+  HANDLE_TH_ERRORS;
+  PyObject* input_objs;
+  PyObject* output_objs;
+  const char *accepted_kwargs[] = {"inputs", "outputs", NULL};
```

Comment:
So is it this function like "re-execute part of the graph between inputs and outputs"?